media-source-respec.html

```diff
@@ -224,7 +221,7 @@ <h3>Definitions</h3>
           <dd><p>The parent media source of a <a>SourceBuffer</a> object is the <a>MediaSource</a> object that created it.</p></dd>
 
           <dt><dfn id="presentation-start-time">Presentation Start Time</dfn></dt>
-          <dd><p>The presentation start time is the earliest time point in the presentation and specifies the <a def-id="videoref" name="initial-playback-position">initial playback position</a> and <a def-id="videoref" name="earliest-possible-position">earliest possible position</a>. All presentations created using this specification have a presentation start time of 0.</p>
+          <dd><p>The presentation start time is the earliest time point in the presentation and specifies the <a def-id="videoref" name="current-playback-position">playback position</a> and <a def-id="videoref" name="earliest-possible-position">earliest possible position</a>. All presentations created using this specification have a presentation start time of 0.</p>
```

Comment:
current-playback-position changes during a playback. MSE prescribes both the earliest-possible-position and the initial-playback-position to both be 0. The linked HTMLMediaElement spec still references "initial playback position" in steps 10 and 11 of the "Once enough of the media data has been fetched to determine the duration of the media resource, its dimensions, and other metadata" in its media data processing steps list.

current-playback-position would be incorrect here. If we must update due a broken link for initial-playback-position, we could file an issue to get that concept exported from HTMLMediaElement for our use and meanwhile explicitly describe the linkage here as being the "initial playback position" for a MediaSource media resource as needed by the "media data processing steps" of HTMLMediaElement.

Reply:
Oops, I forgot to add "initial" back in the prose I changed but today I don't have anything better to offer than putting that adjective outside of the reference and referencing "current playback position", as in:
```html
the initial <a def-id="videoref" name="current-playback-position">playback position</a>
```

The steps 10 and 11 that you mention are the only places in HTML that mention "initial playback position". That concept as a whole no longer exists in HTML (it used to). That actually needs fixing in HTML, because "initial playback position" is wrapped in a `<var>` but that variable is not defined anywhere.

In other words, it's not only a matter of exporting a definition, it's a matter of re-introducing one that got dropped, or considering that "initial" is just common sense to mean the initial value of the "current playback position". I propose the latter for now until that gets improved in HTML.

Reply:
Filed an issue on the HTML spec at https://github.com/whatwg/html/issues/7129